### ISSUE TITLE
Splash v3.4.13 will not install with npm / yarn

### ISSUE THREAD

**Issue by luastoned** [splash-cli/splash-cli#77]
#### Basic informations
- **Splash CLI** version: `3.4.13`
- Node version: `v12.18.3`
- Your OS: **Ubuntu 20.04.1 LTS**

#### Before you go
- [x] I've read the [FAQ](https://github.com/splash-cli/splash-cli/blob/master/FAQ.md) section.
- [ ] I've the latest version of **Splash CLI installed**
- [x] I've searched other issues like mine but I haven't found a solution.

# The ISSUE
Splash CLI will not install via `npm` / `yarn`

**To Reproduce**
```
> yarn global add splash-cli
yarn global v1.22.4
[1/4] Resolving packages...
[2/4] Fetching packages...
[3/4] Linking dependencies...
[4/4] Building fresh packages...
success Installed "splash-cli@3.4.13" with binaries:
      - splash
Done in 5.61s.

> splash
Command 'splash' not found, but can be installed with:

apt install splash
```

```
> npm install --global splash-cli
npm ERR! code ENOENT
npm ERR! syscall chmod
npm ERR! path ~/.nvm/versions/node/v12.18.3/lib/node_modules/splash-cli/build/bin/index.js
npm ERR! errno -2
npm ERR! enoent ENOENT: no such file or directory, chmod '~/.nvm/versions/node/v12.18.3/lib/node_modules/splash-cli/build/bin/index.js'
npm ERR! enoent This is related to npm not being able to find a file.
```

**Expected behavior**
`splash` being available after installing.

**Comment by Rawnly** [comment]
Thanks for reporting you can install the previous version of splash-cli via `npm i -g splash-cli@3.4.12`

**Comment by AlecRust** [comment]
Same errors as above with latest Node v14.10.0.

**Comment by nnadeau** [comment]
confirm that `npm i -g splash-cli@3.4.12` works with `Ubuntu 20.04.1 LTS`, `Node.js v14.12.0`, `npm@6.14.8`

**Comment by Rawnly** [comment]
Thanks @nnadeau 

Unfortunately I didn't found the time  release an hot fix (hopefully soon)
 ✌️

**Comment by Rawnly** [comment]
Fixed in `v3.4.14` Please update to the latest release.
